pkg/apis/eventing/v1alpha1/broker_validation_test.go

```diff
@@ -302,121 +301,3 @@ func TestValidate(t *testing.T) {
 		})
 	}
 }
-
-func TestValidateFunc(t *testing.T) {
-	tests := []struct {
-		name     string
-		b        *unstructured.Unstructured
-		original *eventingv1.Broker
-		want     *apis.FieldError
-	}{{
-		name: "not my broker class",
```

Comment:
Each of these test cases are covered in the test above. This was testing the wrapper `ValidateBroker` function which is now fully removed.